src/pages/community-partners.astro

```diff
@@ -30,4 +30,6 @@ const sponsors = await getCollection("sponsors", ({ data }) => {
       ))}
     </div>
   )}
-</Layout>
+  <p class="pb-5">
+  Are you a community organiser interested in partnering with us? Feel free to reach out at board@europython.eu
```

Comment:
something like this? Mostly to encourage new communities at least to connect with EPS. 
```suggestion
  Are you a Python Community organiser, interested in knowing more about EuroPython Society? Feel free to reach out at board@europython.eu!
```